atuin-server-postgres/src/lib.rs

```diff
@@ -1,14 +1,18 @@
+
+
 use async_trait::async_trait;
```

Comment:
```suggestion
use async_trait::async_trait;
```

you don't need to artificially inflate your line count